MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Clock.xaml

```diff
@@ -547,4 +759,2919 @@
+                                                   Width="24" TextAlignment="Center"/>
+
+                                        <Grid Width="96" Height="80" x:Name="PART_MinuteReadOut">
+                                            <Border x:Name="PART_MinuteReadOutBorder"
+                                                    CornerRadius="{TemplateBinding CornerRadius}">
+                                                <Border.Style>
+                                                    <Style TargetType="Border">
+                                                        <Setter Property="Background" Value="{DynamicResource PrimaryHueMidBrush}"/>
+                                                        <Style.Triggers>
+                                                            <DataTrigger Binding="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=DisplayMode}" Value="{x:Static wpf:ClockDisplayMode.Minutes}">
+                                                                <Setter Property="Background" Value="{DynamicResource SecondaryHueMidBrush}" />
+                                                            </DataTrigger>
+                                                        </Style.Triggers>
+                                                    </Style>
+                                                </Border.Style>
+                                            </Border>
+                                            <TextBlock Text="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=Time, Mode=OneWay, StringFormat={}{0:mm}}"  
+													   x:Name="PART_MinuteReadOutText"
+                                                       HorizontalAlignment="Center"
+                                                       VerticalAlignment="Center">
+                                                <TextBlock.Style>
+                                                    <Style BasedOn="{StaticResource TimeTextBlock}" TargetType="{x:Type TextBlock}">
+                                                        <Style.Triggers>
+                                                            <DataTrigger Binding="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=DisplayMode}" Value="{x:Static wpf:ClockDisplayMode.Minutes}">
+                                                                <Setter Property="Foreground" Value="{DynamicResource SecondaryHueMidForegroundBrush}" />
+                                                            </DataTrigger>
+                                                        </Style.Triggers>
+                                                    </Style>
+                                                </TextBlock.Style>
+                                            </TextBlock>
+                                        </Grid>
+
+                                        <TextBlock x:Name="PART_SecondColonPrefix" Text=":"
```

Comment:
It would be good to use a [string format specifier](https://docs.microsoft.com/en-us/dotnet/standard/base-types/custom-date-and-time-format-strings#timeSeparator) to get a localized version of the time separator.

Reply:
I left it like it was in the current version tbh, didn't even think of it. I tried right now to make it culture responsive, but I'm having some trouble. 
I've had the best luck by using StringFormat on the DateTime property, but it doesn't really work like I'd like to. This doesn't work:

`Text="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=Time, Mode=OneWay, StringFormat={}{0::}}"`

It seems like at least two characters are required for StringFormat to work. Adding a leading or trailing space seems to work though.
On the internet there doesn't seem to be much about this issue. Do you know of something I'm not aware of? Otherwise the best solution I've come to would be to add both a leading and trailing whitespace, which would result in only slightly larger space taken up by the separators. Essentially like this:

`StringFormat={}{0: : }`

Reply:
This implementation seems to work pretty ok. Please let me know if you think a better one could be done

```
<TextBlock Text="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=Time, Mode=OneWay, StringFormat={}{0: : }}"
           Style="{StaticResource TimeTextBlock}"
           Foreground="{DynamicResource MaterialDesignBody}"
           Margin="-6 0 -6 0"
           VerticalAlignment="Center" />
```



![immagine](https://user-images.githubusercontent.com/58171461/120197348-24ada080-c221-11eb-8e79-b73e2e817d3d.png)